Utils.ps1

```diff
@@ -22,6 +22,9 @@ Set-Alias ?? Invoke-NullCoalescing -Force
 function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     $currentEncoding = [Console]::OutputEncoding
     try {
+        # A native executable that writes to stderr AND has its stderr redirected will generate non-terminating
+        # error records if the user has set $ErrorActionPreference to Stop. Override that value in this scope.
+        $ErrorActionPreference = 'Continue'

```

Comment:
Indeed!  And #109 is what I was talking about - diff between console and ISE.  I could go back to searching for .git the old way without calling `git rev-parse`.  I just don't know how to identify a bare repo.  Any tips?